### PR TITLE
Feat: Checkbox colors

### PR DESCRIPTION
**Pull request by MaximeDion-Work**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Feature

## Description

Applied right color resources

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] Tested UWP
- [x] Tested iOS
- [x] Tested Android
- [x] Tested WASM
- [x] Contains **No** breaking changes

- If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: src/library/Uno.Material/Styles/Application/Colors.xaml [hunk 1]
```diff
@@ -22,6 +22,13 @@
 			<Color x:Key="SurfaceColor">#FFFFFF</Color>
 			<Color x:Key="ErrorColor">#F85977</Color>
 			<Color x:Key="OnPrimaryColor">#FFFFFF</Color>
+			<Color x:Key="OnPrimaryMediumColor">#4AFFFFFF</Color>
```
Comment (MatFillion) [inline src/library/Uno.Material/Styles/Application/Colors.xaml:25]: Je viens juste de les enlever -_-

Comment (MaximeDion-Work) [reply src/library/Uno.Material/Styles/Application/Colors.xaml:25]: What do I use for SelectControlToggleOnBackgroundColor then?

File: src/library/Uno.Material/Styles/Application/Colors.xaml [hunk 2]
```diff
@@ -186,6 +193,31 @@
 	<SolidColorBrush x:Key="SecondaryColor02Brush"
 					 Color="{StaticResource SecondaryColor02}" />
 
+	
+	<SolidColorBrush x:Key="SelectControlBackgroundColor"
```
Comment (MatFillion) [inline src/library/Uno.Material/Styles/Application/Colors.xaml:197]: We should only keep the color palette here, control specific colors should go in their own resource dictionary. In this case they might be shared between 2-3 controls so I suggest a resourceDictionary SelectionControlColors.xaml and merge it where you need it

File: src/library/Uno.Material/Styles/Controls/CheckBox.xaml [hunk 3]
```diff
@@ -179,40 +194,42 @@
...
+								  Fill="{ThemeResource SelectControlIndicatorColor}"
 								  VerticalAlignment="Center"
 								  HorizontalAlignment="Center"
 								  Stretch="Uniform"
 								  Width="12"
 								  Height="12"
 								  Opacity="0"
-								  AutomationProperties.AccessibilityView="Raw" />
+								  AutomationProperties.AccessibilityView="Raw"
+								  xamarin:Margin="0,2,0,0" />
```
Comment (MatFillion) [inline src/library/Uno.Material/Styles/Controls/CheckBox.xaml:232]: On devrait pas avoir besoin de ça. C'est pourquoi?

Comment (MaximeDion-Work) [reply src/library/Uno.Material/Styles/Controls/CheckBox.xaml:232]: les icons de check et - sont pas bien aligner sur ios et android.

Comment (MatFillion) [reply src/library/Uno.Material/Styles/Controls/CheckBox.xaml:232]: Ok, please open github issue just so we remember

**Pull request merged**